### PR TITLE
[WIP] Add basic full sync info + Add Snapshot Full sync for 2000k + height 

### PR DESCRIPTION
**Pull request by mambisi**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Shows test info on CI
Full sync test for 2000k+ snapshots

### CODE REVIEW

File: ci/parallel_sync/sync_then_diff.sh [hunk 1]
```diff
@@ -35,6 +36,19 @@ setup_vars() {
   NODE_RESTARTS=0
 }
 
+echo "======== Sync Test Info ==========
+- Block range: ${START_BLOCK} - ${STOP_BLOCK}
+- Base snapshot: grc.io/br-blockchains-dev/datadir-${START_BLOCK}
```
Comment (prasannavl) [inline ci/parallel_sync/sync_then_diff.sh:41]:
```suggestion
- Base snapshot: https://grc.io/br-blockchains-dev/datadir-${START_BLOCK}
```

File: ci/parallel_sync/sync_to_tip.sh [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  BLOCK=0
+  ATTEMPTS=0
+  START_BLOCK=${START_BLOCK:-0}
+  STOP_BLOCK=${STOP_BLOCK:-0}
+  MAX_ATTEMPTS=10
+  MAX_NODE_RESTARTS=5
+  NODE_RESTARTS=0
+}
+
+echo "======== Sync Test Info ==========
```
Comment (prasannavl) [inline ci/parallel_sync/sync_to_tip.sh:38]: Move this into a function and call it from main, so its cleaner.

File: ci/parallel_sync/sync_to_tip.sh [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+- Block range: ${START_BLOCK} - ${STOP_BLOCK}
+- Base snapshot: https://gcr.io/br-blockchains-dev/datadir-${START_BLOCK}
```
Comment (prasannavl) [inline ci/parallel_sync/sync_to_tip.sh:40]: Can we get a direct link for this? I assume this leads to a dir.

File: ci/parallel_sync/sync_then_diff.sh [hunk 4]
```diff
@@ -30,11 +31,26 @@ setup_vars() {
...
+- Reference logs:
+  - debug.log: $REF_LOG_PATH
+- Commands used:
+ - $ACCOUNT_BALANCES_CMD
+ - $LIST_ANCHORS_CMD
+- defid cmd: ${DEFI_CLI_CMD}
+- defi.conf:
+  $(cat "$CONF_FILE")
+----------------------------------
+"
```
Comment (prasannavl) [inline ci/parallel_sync/sync_then_diff.sh:53]: Similar adaptations to the other code block

File: ci/parallel_sync/sync_to_tip.sh (hunk 3 above)
Comment (mambisi) [reply ci/parallel_sync/sync_to_tip.sh:40]: its the link to snapshot(tar) file

**Comment by Jouzo** [comment]
Superseded by #1596 and #1523

**Pull request closed without merging**